### ISSUE TITLE
Suggest actions to take

### ISSUE THREAD

**Issue by dneuhaeuser-zalando** [hjacobs/kube-resource-report#14]
> You can potentially save X USD every month by optimizing resource requests and reducing slack.

How? It would be good to have a link there suggesting, if not specific, then at least general options available.

**Comment by hjacobs** [comment]
Good point, the main driver is adapting CPU/memory requests to match actual usage (requests - usage * cost_per_unit is the formula to get "slack costs").
